platform/vendor/vendor.yaml

```diff
@@ -14,9 +14,9 @@ files:
     destination_dir: "tekton/dashboard"
     version: "v0.24.1"
     validation_type: "sha256"
-  - release_file: "https://raw.githubusercontent.com/kyverno/kyverno/5b4d4c266353981a559fe210b4e85100fa3bf397/config/install.yaml"
+  - release_file: "https://raw.githubusercontent.com/kyverno/kyverno/346a7c41c8041f03c553690eb2ab61d2db3e8742/config/install.yaml"
     destination_dir: "kyverno/release"
-    sha256: "36040e11630c4730df1a279ecc29ac6aabfd03b7b3790fc192b574b4125b56d0"
+    sha256: "228d16212a422d7953493f6b44cb27ed55685372a2ac87f5911557b01b755b17"
     validation_type: "sha256"
     version: "v1.6.0"
```

Comment:
Probably want to update this. It's not actually used for `sha256` type, but gives the user and idea of what version is being referred to. Especially since the version is not in the URL for the release file.